esmvaltool/diag_scripts/hydrology/lazy_regrid.py

```diff
@@ -1,20 +1,17 @@
 """Lazy regridding, because this is not supported by iris (yet).
 
 Iris issue requesting the feature:
-https://github.com/SciTools/iris/issues/3700
+https://github.com/SciTools/iris/issues/3808
 """
-import copy
-
-import iris
 import numpy as np
-
-HORIZONTAL_SCHEMES = {
-    'linear': iris.analysis.Linear(extrapolation_mode='mask'),
-    'linear_extrapolate':
-    iris.analysis.Linear(extrapolation_mode='extrapolate'),
-    'nearest': iris.analysis.Nearest(extrapolation_mode='mask'),
-    'area_weighted': iris.analysis.AreaWeighted(),
-}
+from esmvalcore.preprocessor import regrid
+
+HORIZONTAL_SCHEMES = [
+    'linear',
+    'linear_extrapolate',
+    'nearest',
+    'area_weighted',
+]
```

Comment:
This can be removed too, the `esmvalcore.preprocessor.regrid` function already checks that the scheme is valid (the original code was copied from there).

Reply:
It is removed.